cxotime/utils.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import astropy.units as u
+import numpy as np
+
+from cxotime import CxoTime, CxoTimeLike
+
+
+def get_range_in_chunks(start: CxoTimeLike, stop: CxoTimeLike, dt_max: u.Quantity):
```

Comment:
Maybe this would be best as a `CxoTime` class method `linspace`. This would accept `start, stop, num: int | None, step_max: u.Quantity | None`. This borrows from numpy `linspace` and also `step` from `arange`. So a call would look like:
```
from cxotime import CxoTime
times = CxoTime.linspace(start, stop, step_max=1 * u.hr)
```